### ISSUE TITLE
Clean/Clarify use of icons in conversation editor

### ISSUE THREAD

**Issue by willnationsdev** [coppolaemilio/dialogic#4]
There are 3 distinct things that appear with the script icon:

1. The button to display the conversation editor within the Dialogic main screen.
1. The texture shown beside the names of individual conversations within the conversation editor.
1. The button to add an actual GDScript to the conversation.

![conversation_editor_edited](https://user-images.githubusercontent.com/16217563/95007919-9f62e500-05da-11eb-9090-8c6a35cf89f7.png)

There should be distinct icons for each of these purposes, imo.

**Comment by coppolaemilio** [comment]
Yes, I need to make a different icons for those. The A one would be the timeline icon, B should be the dialogue button or something like that and C I think it can stay like that, since it is going to be the same as adding a script to your game, so it makes sense that it stays as the default godot one.
